bundles/org.openhab.binding.miele/src/main/resources/OH-INF/thing/channeltypes.xml

```diff
@@ -86,98 +90,50 @@
 		<state readOnly="true"></state>
 	</channel-type>
 
-	<channel-type id="target" advanced="true">
-		<item-type>Number</item-type>
-		<label>Target Temperature</label>
-		<description>Target temperature to be reached by the oven</description>
-		<state readOnly="true"></state>
-	</channel-type>
-
-	<channel-type id="measured" advanced="true">
-		<item-type>Number</item-type>
-		<label>Measured Temperature</label>
-		<description>Actual measured temperature in the oven</description>
-		<state readOnly="true"></state>
-	</channel-type>
-
-	<channel-type id="temp" advanced="true">
-		<item-type>Number</item-type>
+	<channel-type id="temperature">
```

Comment:
Defining so generic channel type is not recommended.
At least, you could have `temperature` and `TargetTemperature` which are two very different notions.

Reply:
I considered it, and it would make sense if target temperature was actually settable. Unfortunately it's not, so decided to go with shared definition as all properties are identical, so defining two different types seems arbitrary and redundant. Arbitrary because previously we had a channel type for each individual channel. Having two is better than having nine, but still - why two?

Reply:
@kaikreuzer : please comment, I remember you told me to not do that (defining ulmtra generic chan,nel types like "temperature") a very long time ago. Is it still the case ?

Reply:
You convinced me to split into current and target for reuse of same label - fixed. Still interested in comments about finding right balance between "too generic" and "too redundant". :-)